codecoverage.py

```diff
@@ -76,9 +76,15 @@ def suite_name_from_task_name(name):
     return '-'.join(parts)
 
 
-def download_coverage_artifacts(build_task_id, suites):
+def download_coverage_artifacts(build_task_id, suites, suites_to_ignore):
```

Comment:
You can have a default for `suites_to_ignore`:
```
def download_coverage_artifacts(build_task_id, suites, suites_to_ignore=['talos', 'awsy']):
```

Reply:
But then ['talos', 'awsy'] will be overridden if other suites would be given as command line arguments? I thought, 'talos' and 'awsy' should always be added to ignored suites.

Reply:
No, they should be ignored by default, but the user should be able to overwrite the default.

Reply:
There might be cases where the user is interested to know the coverage given by talos or awsy.